fastlane_bot/events/utils.py

```diff
@@ -1716,6 +1716,27 @@ def handle_target_token_addresses(static_pool_data: pd.DataFrame, target_tokens:
     return target_token_addresses
 
 
+def verify_min_bnt_is_respected(bot: CarbonBot, mgr: Any):
```

Comment:
Never used

Reply:
Replaced this PR with https://github.com/bancorprotocol/fastlane-bot/pull/632 since some merge mistakes were made